docs/conf.py

```diff
@@ -1,43 +1,25 @@
-"""
-Configuration file for the Sphinx documentation builder.
-"""
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
 
 import datetime
 import os
 import pathlib
-import warnings
-from pathlib import Path
 
-from astropy.utils.exceptions import AstropyDeprecationWarning
-from matplotlib import MatplotlibDeprecationWarning
-from packaging.version import Version
-from sunpy.util.exceptions import SunpyDeprecationWarning, SunpyPendingDeprecationWarning
-from sunpy_sphinx_theme.conf import *  # NOQA: F403
+# -- Project information -----------------------------------------------------
 
+# The full version, including alpha/beta/rc tags
 from sunkit_image import __version__
 
-# -- Read the Docs Specific Configuration --------------------------------------
-on_rtd = os.environ.get("READTHEDOCS", None) == "True"
-if on_rtd:
-    os.environ["SUNPY_CONFIGDIR"] = "/home/docs/"
-    os.environ["HOME"] = "/home/docs/"
-    os.environ["LANG"] = "C"
-    os.environ["LC_ALL"] = "C"
-    os.environ["HIDE_PARFIVE_PROGRESS"] = "True"
```

Comment:
I think we want to discuss adding this to the template?

Reply:
Yeah i've kept it very minimal for now docs wise.

I'll put this back in for now